### ISSUE TITLE
Windows: `repeat_report` fails due to file descriptor being left open

### ISSUE THREAD

**Issue by zatkins-dev** [quaquel/pyNetLogo#69]
The `tempfile.mkstemp` function opens and returns a file descriptor which is currently being discarded in `NetLogoLink.repeat_report`. This is "fine" for Linux/MacOS as calls to `os.remove` on a file descriptor which is being used by another process are silently queued, but on Windows, this results in an exception being thrown.

The easy solution is to close the file descriptor by changing this (core.py lines 512-519):
```python
        for variable in cols:
            _, fn = tempfile.mkstemp(suffix=".txt", dir=tempfolder)
            fns[variable] = fn
            fn = '"{}"'.format(fn)
            fn = fn.replace(os.sep, "/")

            nc = r"{0} {1} {2} {3}".format("file-open", fn, "file-write", variable)
            commands.append(nc)
```
to this:
```python
        for variable in cols:
            fd, fn = tempfile.mkstemp(suffix=".txt", dir=tempfolder)
            os.close(fd)
            fns[variable] = fn
            fn = '"{}"'.format(fn)
            fn = fn.replace(os.sep, "/")

            nc = r"{0} {1} {2} {3}".format("file-open", fn, "file-write", variable)
            commands.append(nc)
```
where the latter immediately closes the file descriptor to allow for successful deletion of the file at the end of the function.

**Comment by EwoutH** [comment]
Thanks for reporting this bug! Since you already have a proposed fix, are you willing to open a Pull request for it?

**Comment by quaquel** [comment]
I have released a new version 0.5.1 which contains a bugfix for this problem

**Comment by zatkins-dev** [comment]
Thank you! Sorry, I got caught up with other projects and didn't get a chance to put in that PR.
